rules/cross-platform/privilege_escalation_sudo_buffer_overflow.toml

```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2021/02/03"
+maturity = "production"
+updated_date = "2021/02/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the attempted use of a heap-based buffer overflow vulnerability for the Sudo binary in Unix-like systems
+(CVE-2021-3156). Successful exploitation allows an unprivileged user to escalate to the root user.
+"""
+false_positives = [
+    """
+    This rule could generate false positives if the process arguments leveraged by the expoit are shared by custom
+    scripts using the Sudo or Sudoedit binaries. Only Sudo versions 1.8.2 through 1.8.31p2 and 1.9.0 through
+    1.9.5p1 are affected; if those versions are not present on the endpoint, this could be a false positive.
+    """,
+]
+from = "now-9m"
```

Comment:
this is usually set for EQL rules based on the timing of the data relaying. For KQL we usually just stick with the default.
```suggestion

```

Reply:
we actually need it for anything from endpoint sensor due to the timing, so we should be good to leave it here

Reply:
but this does impact the threshold - 100/9mins - is that what is desired

Reply:
there's lots of rules that have `logs-endpoint.events.*` without the 9m lookback, so that may be worth a bulk change

Reply:
The interval should still be the default `5m`, right?

Reply:
yup, currently the rule will run every 5m, and then look back an additional 4m upon each execution